Containers/talk-recording/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.11.3-alpine3.18
+
+COPY --chmod=775 start.sh /start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        xvfb \
+        ffmpeg \
+        firefox \
```

Comment:
```suggestion
        firefox-esr \
```

Reply:
Not sure if esr is better
https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html

Reply:
Not sure either. Lets try normal firefox first?

Reply:
yes